src/app/modules/submission/encrypt-submission/IncomingEncryptSubmission.class.ts

```diff
@@ -37,7 +37,11 @@ export default class IncomingEncryptSubmission extends IncomingSubmission {
     ProcessingError | ConflictError | ValidateFieldError[]
   > {
     return checkIsEncryptedEncoding(encryptedContent)
-      .andThen(() => getFilteredResponses(form, responses))
+      .andThen(() => {
+        const newEncryptionBoundary: boolean = form.get('newEncryptionBoundary')
+        if (newEncryptionBoundary) return ok(responses as FilteredResponse[])
+        else return getFilteredResponses(form, responses)
+      })
```

Comment:
to add unit tests for this

Reply:
added in [a61ab60](https://github.com/opengovsg/FormSG/pull/6595/commits/a61ab60ac5395dbefa077fe34cc6a424efbdca0e)